Tests/SentryTests/Integrations/SessionReplay/SentryBreadcrumbReplayConverterTests.swift

```diff
@@ -0,0 +1,162 @@
+        XCTAssertEqual(eventPayload?["category"] as? String, "app.foreground")
+    }
+    
+    func testNavigationBreadcrumbOrientation() {
+        let sut = SentryBreadcrumbReplayConverter()
+        let crumb = Breadcrumb(level: .info, category: "device.orientation")
+        crumb.type = "navigation"
+        crumb.data = ["position": "portrait"]
+        let result = sut.replayBreadcrumbs(from: [crumb])
+        
+        XCTAssertEqual(result.count, 1)
+        let event = result.first?.serialize()
+        let eventData = event?["data"] as? [String: Any]
+        let eventPayload = eventData?["payload"] as? [String: Any]
+        let payloadData = eventPayload?["data"] as? [String: Any]
+        
+        XCTAssertEqual(event?["type"] as? Int, 5)
+        XCTAssertEqual(eventData?["tag"] as? String, "breadcrumb")
+        XCTAssertEqual(eventPayload?["category"] as? String, "device.orientation")
+        XCTAssertEqual(payloadData?["position"] as? String, "portrait")
+    }
+    
+    func testNavigationBreadcrumbNavigate() {
+        let sut = SentryBreadcrumbReplayConverter()
+        let crumb = Breadcrumb(level: .info, category: "ui.lifecycle")
+        crumb.type = "navigation"
+        crumb.data = ["screen": "TestViewController"]
+        let result = sut.replayBreadcrumbs(from: [crumb])
+        
+        XCTAssertEqual(result.count, 1)
+        let event = result.first?.serialize()
+        let eventData = event?["data"] as? [String: Any]
+        let eventPayload = eventData?["payload"] as? [String: Any]
+        let payloadData = eventPayload?["data"] as? [String: Any]
+        
+        XCTAssertEqual(event?["type"] as? Int, 5)
+        XCTAssertEqual(eventData?["tag"] as? String, "breadcrumb")
+        XCTAssertEqual(eventPayload?["category"] as? String, "navigation")
+        XCTAssertEqual(payloadData?["to"] as? String, "TestViewController")
+    }
+    
+    func testHttpBreadcrumb() throws {
+        let sut = SentryBreadcrumbReplayConverter()
+        let breadcrumb = Breadcrumb(level: .info, category: "http")
+        
+        breadcrumb.data = [
+            "url": "https://test.com",
+            "method": "GET",
+            "response_body_size": 1_024,
+            "http.query": "query=value",
+            "http.fragment": "frag",
+            "status_code": 200
+        ]
+        
+        let result = try XCTUnwrap(sut.replayBreadcrumbs(from: [breadcrumb]).first)
+        let crumbData = try XCTUnwrap(result.data)
+        let payload = try XCTUnwrap(crumbData["payload"] as? [String: Any])
+        let payloadData = try XCTUnwrap(payload["data"] as? [String: Any])
+        
+        XCTAssertEqual(result.timestamp, breadcrumb.timestamp)
+        XCTAssertEqual(crumbData["tag"] as? String, "performanceSpan")
+        XCTAssertEqual(payload["description"] as? String, "https://test.com")
+        XCTAssertEqual(payload["op"] as? String, "resource.http")
```

Comment:
`m`: Also asserting the `startTimesamp` and `endTimestamp` properties of the payload improves this test case, as no other test validates these properties.